core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -1315,4 +1318,324 @@ class UserServiceSpec
       result should contain theSameElementsAs expected
     }
   }
+
+  behavior of "addUserToBillingProjectV2"
+
+  it should "update billing profile record when a user is added to a billing project" in {
+    val billingProfileId = UUID.randomUUID()
```

Comment:
Since these test points no longer require `withMinimalTestDatabase`, I might want to move them into a separate test class so they don't have the setup overhead.